lib/main/STM32G4/Middlewares/ST/STM32_USB_Device_Library/Class/MSC/Src/usbd_msc.c

```diff
@@ -296,6 +297,17 @@ __ALIGN_BEGIN  uint8_t USBD_MSC_DeviceQualifierDesc[USB_LEN_DEV_QUALIFIER_DESC]
 uint8_t  USBD_MSC_Init (USBD_HandleTypeDef *pdev, uint8_t cfgidx)
 {
   UNUSED(cfgidx);
+  USBD_MSC_BOT_HandleTypeDef *hmsc;
+
+  hmsc = &ClassData;
+
+  if (hmsc == NULL)
```

Comment:
I think this is pointless check since `hsmc` pointer cannot be zero (point to static object and initialized just above). 
Or is this typo?

Reply:
I think this is for safety right？

Reply:
Safety if it checks `if (pdev == NULL)` :)

Reply:
All valid points. `pdev` isn't checked on other routines in this library, so I'll not add it for this, but I've removed the pointless NULL check.